src/main/java/decodex/storage/Storage.java

```diff
@@ -24,32 +24,28 @@
 // @@author Kair0s3
 public class Storage {
 
-    /**
-     * Specifies the default directories for input, output and recipes respectively.
-     */
+    // Specifies the default directories for input, output and recipes respectively.

```

Comment:
Did not see, perhaps just state that it is for `recipes`? We do not want to give early spoilers yet

Reply:
The following commits itself is a spoiler 😂 

*SPOILER - If you see this, please take precautions before viewing the following commits, they are literally spoilers for the future of `Decodex`*